src/relations/db.py

```diff
@@ -257,8 +257,9 @@ def _on_relation_joined(self, join_event: RelationJoinedEvent):
             logger.info(init_msg)
 
             self.charm.backend.postgres.create_user(user, password, admin=self.admin)
-            self.charm.backend.postgres.create_database(database, user)
-
+            self.charm.backend.postgres.create_database(
+                database, user, client_relations=self.charm.client_relations
+            )
```

Comment:
Copied from the PG charm. This can potentially cause issues if both PG and PGB are related on the same database, as both will think they own it.

Reply:
I think it may happen only when we change the PgBouncer extra user roles from `SUPERUSER` to fewer privileges.